### PR TITLE
changes in Organization name some validation changes

### PR DESCRIPTION
**Pull request by JyotiJaiswal169**

### CODE REVIEW

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { OrganisationUnit } from "@domain/index";
+import { QueryRunner } from "typeorm";
+
+import { BaseSeed } from "../tools/bases/base-seed";
+
+export class alterOrganisationsUnitsNames1640858698485 extends BaseSeed {
+  public name = "alterOrganisationsUnitsNames1640858698485";
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    const data = [
```
Comment (assimoes-bjss) [inline __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:12]: Why are you not using the `queryRunner` object?
Why using the repo on here? Is there a reason to not write plain T-SQL statements?

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      const orgUnitObj = data[idx];
+
+      const filterOptions = {
+        where: { acronym: orgUnitObj.acronym },
+      };
+      const orgUnit = await orgUnitRepo.findOne(filterOptions);
+
+      if (!orgUnit) continue;
+
+      orgUnit.name = orgUnitObj.name;
```
Comment (assimoes-bjss) [inline __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:36]: Isn't this updating the the name by exactly the same value? Like...doing nothing?

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      await orgUnitRepo.save(orgUnit);
+    }
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<any> {
+    // const repo = this.getRepository();
```
Comment (assimoes-bjss) [inline __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:42]: How can I revert this?

**Review by assimoes-bjss — CHANGES_REQUESTED** [review]
Why not use

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts (hunk 1 above)
Comment (JyotiJaiswal169) [reply __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:12]: I have used existing seed code and created new seed to update Organization name in database 
This (1627914973606-alter-organisation-units-names.ts
 )seed file to update Organization name was already there

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts (hunk 2 above)
Comment (JyotiJaiswal169) [reply __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:36]: On local i have tested its updating the name

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts (hunk 1 above)
Comment (assimoes-bjss) [reply __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:12]: Ok.

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts (hunk 2 above)
Comment (assimoes-bjss) [reply __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:36]: Ok.

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts (hunk 3 above)
Comment (assimoes-bjss) [reply __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:42]: Given the above comments, there needs to be a way of reverting these changes. Regardless the fact there is a seed file created on the 2nd of August that does not have a revert.

**Review by assimoes-bjss — CHANGES_REQUESTED** [review]

File: __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts (hunk 3 above)
Comment (JyotiJaiswal169) [reply __external__/domain/seeds/1640858698485-alter-organisations-units-names.ts:42]: Okay

**Review by assimoes-bjss — APPROVED** [review]

**Review by BJSS-RicardoTavares — APPROVED** [review]

**Pull request merged**